vcf.c

```diff
@@ -1318,6 +1318,13 @@ static const char *get_type_name(int type) {
     return types[t];
 }
 
+static void bcf_record_check_err(const bcf_hdr_t *hdr, bcf1_t *rec, char *msg,
+                                 uint32_t *reports, int i, const char *s) {
+    if (*reports == 0 || hts_verbose >= HTS_LOG_DEBUG)
+        hts_log_warning(msg, msg, bcf_seqname_safe(hdr,rec), rec->pos+1, i, s);
```

Comment:
Too many `msg` here.  It would also be better to move the format string from the caller into here given that it's only used for "Invalid FORMAT (id|type)".  For the invalid type case, `get_type_name()` will only return "null" or "unknown" as anything else is valid, so removing that part of the message would not be a great loss.